tf/env/local/cluster.tf

```diff
@@ -0,0 +1,9 @@
+resource "kubernetes_config_map" "cluster" {
+  metadata {
+    name = "cluster-config-map"
+  }
+
+  data = {
+    ipv4_range = "10.0.0.0/8"
```

Comment:
seems easiest to hardcode this here. I guess we could probably pull this from minikube somehow but hardly worth the effort